config/notifiers.go

```diff
@@ -459,17 +460,38 @@ type OpsGenieConfig struct {
 	Description string            `yaml:"description,omitempty" json:"description,omitempty"`
 	Source      string            `yaml:"source,omitempty" json:"source,omitempty"`
 	Details     map[string]string `yaml:"details,omitempty" json:"details,omitempty"`
-	Teams       string            `yaml:"teams,omitempty" json:"teams,omitempty"`
-	Tags        string            `yaml:"tags,omitempty" json:"tags,omitempty"`
-	Note        string            `yaml:"note,omitempty" json:"note,omitempty"`
-	Priority    string            `yaml:"priority,omitempty" json:"priority,omitempty"`
+	// Teams field is deprecated.
```

Comment:
No strong feeling about this one but I'd rather remove the field as part of the change.

Reply:
Up to you guys (: The problem is that we will not fail the config unmarshalling if people will be still on `Teams` (and they will be). Thoughts?

Reply:
The unmarshaling should still fail as we use [`yaml.UnmarshalStrict()`](https://godoc.org/gopkg.in/yaml.v2#UnmarshalStrict).